fplll/nr/matrix.h

```diff
@@ -266,19 +266,24 @@ template <class ZT> class ZZ_mat : public Matrix<Z_NR<ZT>>
   void gen_simdioph(int bits, int bits2);
   /** Generate a random matrix of uniform distribution. */
   void gen_uniform(int bits);
+  /** Generate a random matrix of uniform distribution (non-square flexible). */
```

Comment:
Why not permit non-square for `gen_uniform`? I don't think we need two functions for this.

Reply:
I don't think so either, but when I saw it actually returns an error message when the matrix is not square, I thought there was a specific reason for it and decided to create a new one to be sure I don't potentially break other parts of the code.

I can merge them both in the same function "gen_uniform" indeed.